dango/types/src/lending.rs

```diff
@@ -62,4 +108,10 @@ pub enum QueryMsg {
         start_after: Option<Addr>,
         limit: Option<u32>,
     },
+    /// Returns all collateral powers.
+    #[returns(BTreeMap<Denom, CollateralPower>)]
+    CollateralPowers {},
+    /// Queries the health of a margin account.
+    #[returns(HealthResponse)]
+    Health { account: Addr },
```

Comment:
I think the `Health` query should be in the margin account instead of lending. Lending only concerns about lending out assets and compound interests. The margin account concerns health and liquidation.

Collateral powers probably should be an app config.